src/styles/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
```

Comment:
Consider importing a `_variables.scss` file at the top of this file. This will help in managing common styles and variables, such as colors and dimensions, across different components.